modules/javafx.base/src/test/java/test/util/memory/JMemoryBuddy.java

```diff
@@ -91,6 +92,38 @@ public static void assertCollectable(WeakReference weakReference) {
         }
     }
 
+    /**
+     * Checks whether an array of WeakReference objects can be collected.
```

Comment:
I think it should rather say something like "whether content of all references in the array", rather than array itself.

Reply:
Done (+ others)